This file is part of MXE.
See index.html for further information.

From abe9b3992e846f154f05853a30486139d5bca8d6 Mon Sep 17 00:00:00 2001
From: Tom Molesworth <tom@entitymodel.com>
Date: Mon, 20 Jan 2014 21:23:26 +0100
Subject: [PATCH] Patch texi2pod.pl for perl-5.14+

As described in #200. Similar to #198, this time just needed to
add * bullets to the =item lines to avoid pod errors for the wget
exit codes.

diff --git a/doc/texi2pod.pl b/doc/texi2pod.pl
index 9db6de1..f8f9544 100755
--- a/doc/texi2pod.pl
+++ b/doc/texi2pod.pl
@@ -294,7 +294,7 @@ while(<$inf>) {
                 $_ = "\n=item C<$thing>\n";
             } else {
                 # Entity escapes prevent munging by the <> processing below.
-                $_ = "\n=item $ic\&LT;$thing\&GT;\n";
+                $_ = "\n=item * $ic\&LT;$thing\&GT;\n";
             }
 	} else {
 	    $_ = "\n=item $ic\n";
-- 
1.8.4

